test/wine_o_inventory_api/products_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule WineOInventoryApi.ProductsTest do
+  use WineOInventoryApi.DataCase
+
+  alias WineOInventoryApi.Products
+
+  describe "wines" do
+    alias WineOInventoryApi.Products.Wine
+
+    import WineOInventoryApi.ProductsFixtures
```

Comment:
seems a little weird to have these within a describe block. Is that a new Elixir thing?

Reply:
Not sure if it's a new thing. While I was reading through the documentation on testing in phoenix, they utilize `describe` blocks to group similar tests together. So for example, if we defined a `Soda` schema, tests related to that resource would be grouped together inside a `describe` block within `products_test.exs`

Reply:
I meant more specifically the import statements.